pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -37,6 +37,17 @@ const (
 	// present on a machine and the TopologyManager is enabled, an error will
 	// be returned and the TopologyManager will not be loaded.
 	maxAllowableNUMANodes = 8
+	// containerScopeTopology specifies the TopologyManagerScope per container.
+	containerScopeTopology = "container"
```

Comment:
why is it scopetopology, not topologyscope?

```suggestion
	containerTopologyScope = "container"
```

I'd also question dropping the `Manager` from the name. Why change terminology? Is the shorter name better here?

Reply:
Done

Reply:
> To make it more visible that no more changes were made:
> 
> 1. Diff in the external tool: https://www.diffchecker.com/Ky5HfibH
> 2. Test cases comparison (with trimmed whitespace): https://www.diffchecker.com/HlOFYzU6

This diff makes it much clearer what was done here. Thanks for putting this together.